src/migrations/20171005123329_transfers.ts

```diff
@@ -4,7 +4,7 @@ exports.up = async (knex: Knex): Promise<any> => {
   return knex.schema.dropTableIfExists("transfers").then((): PromiseLike<any> => {
     return knex.schema.raw(`CREATE TABLE transfers (
```

Comment:
it would be really nice to get rid of these .raw(CREATE)'s and turn them into proper create tables

Reply:
Agreed --- maybe another lager cleanup since this patch is kinda unwieldy anyway?